packages/core/src/index.ts

```diff
@@ -65,4 +65,5 @@ export type {
   GriffelRenderer,
 } from './types';
 
+// Private exports, are used by devtools
```

Comment:
maybe prefix with `@internal` and then strip on build and use `stripInternal` in the tsconfig ?

you'll be able to use it with aliases in the monorepo but the types will be stripped on build

Reply:
Didn't know about `stripInternal`, it's a good idea. I will try it in another PR.